src/main/resources/db/migration/V3__create_sales_employee.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE sales_employee(
+    employee_id smallint PRIMARY KEY AUTO_INCREMENT,
+    name varchar(255) NOT NULL,
+    salary decimal(11, 2) NOT NULL,
+    bank_acc_num varchar(34) NOT NULL,
+    ni_num char(9) NOT NULL,
+    comm_rate decimal(5,2) NOT NULL,
+    role_id tinyint NOT NULL
+);
+
+
+INSERT INTO sales_employee(name, salary, bank_acc_num, ni_num, comm_rate, role_id)
```

Comment:
These insert statements could all be in the one statement

Reply:
Fixed the insert statement now.